.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"

```

Comment:
Cool, mangler bare tester som precommit hook

Reply:
Hm.. Litt usikker på om jeg synes at en skal være nødt til å kjøre tester før hver commit. F.eks om en har skrevet tre tester og to feiler, mens en går igjennom betyr ikke det at en ikke vil comitte det som gjør at den ene går i grønt (samme med push). Kunne evt satt opp en github flow som kjører testene når en pull request blir åpna?